packages/mendel-outlet-manifest/package.json

```diff
@@ -17,6 +17,7 @@
     "babel-plugin-transform-inline-environment-variables": "0.0.2",
     "debug": "^2.6.0",
     "mendel-manifest-uglify": "^3.0.2",
+    "mendel-development": "^3.0.1",
```

Comment:
I think moving this to outlet is the future outcome. I just don't want o go through all the refactor right now because of time constraints. `mendel-development` is the grab bag for all cross package utilities since mendel-v1. If more than 1 package uses it. it is there. You can think of it as `mendel-developmente-lib` or `mendel-development-utils`. The other way to fix it is to move a lot of mendel-pipeline inside of mendel-development. 😄